src/lib/services/graphql/graphql.service.ts

```diff
@@ -419,7 +432,9 @@ export class GraphqlService {
   }
 
   private monitorWithOfflineEvent(): void {
-    window.addEventListener('offline', this.sendPingWithThisBound);
+    if (typeof window !== 'undefined') {
```

Comment:
In another location I use globalThis for working with node and browser, but here node will never emit an `"offline"` event on the global object so it's better to be explicit